include/config/config_benchmark.h

```diff
@@ -13,5 +13,5 @@
 #ifdef ENABLE_CREDITS_BENCHMARK
```

Comment:
This is outside the scope of this PR, but the ifdef check being here is potentially problematic for a number of reasons. This should probably be moved into the safeguards file and it needs to account for `DISABLE_ALL`.